openshift/template.yaml

```diff
@@ -143,7 +143,7 @@ parameters:
 - description: Variable which defines time between requests for updates from registries
   displayName: Sleep interval
   name: SLEEP_INTERVAL
-  value: "60"
+  value: "10"
```

Comment:
IIRC 60 minutes was set because people were not happy with tons of logs on production (this was another issue different from logging everything everytime ;)

Reply:
Exactly, this should not have any impact, if we are logging only new versions. (It does not matter if we get 5 logs every 10 minutes or 30 logs once an hour)...